File: Lista4/exec4/livro.py
class CD(estoque):

    #  * expande a classe Estoque com as info do Livro
    #  * @param nome
    #  * @param num
    #  * @param qte
    def __init__(self, nome: str, num: int, qte: int):
        super().__int__(nome, num, qte)

    def __str__(self) -> str:
        s = 'Tipo: Livro'
        s += "Nome: " + self.getNome() + "\n"
        s += "Código de Barras: " + self.getNum() + "\n"
        s += "Quantia em Estoque: " + self.getQte() + "\n"
        return s

File: Lista3/Exec4/src/exec4/Livro.java
/**
 * Classe livro
 */
public class Livro extends Estoque {
    
    /**
     * expande a classe Estoque com as info do livro
     * @param nome
     * @param num
     * @param qte
     */
    public Livro(String nome, long num, int qte){
        super(nome, num, qte);
    }


    @Override
    public String toString(){
        String s = "\nTipo: LIVRO\n";
        s += "Nome: " + getNome() + "\n";
        s += "Código de Barras: " + getNum() + "\n";
        s += "Quantia em Estoque: " + getQte() + "\n";
        return s;
    }

}